sdk/include/opentelemetry/sdk/common/attribute_utils.h

```diff
@@ -42,8 +65,12 @@ struct AttributeConverter
   OwnedAttributeValue operator()(double v) { return OwnedAttributeValue(v); }
   OwnedAttributeValue operator()(nostd::string_view v)
   {
-    return OwnedAttributeValue(std::string(v));
+    return OwnedAttributeValue(std::string(v.data(), v.size()));
   }
+  OwnedAttributeValue operator()(const char *s) { return OwnedAttributeValue(std::string(s)); }
```

Comment:
This is unnecessary, as it's covered by `OwnedAttributeValue operator()(nostd::string_view v)`.

Reply:
Would it be better to keep this one, and remove the `OwnedAttributeValue operator()(nostd::string_view v)`? Seems this one would be more versatile (e.g. supporting the CSTRING TYPE as defined in the macro)

Reply:
I think `OwnedAttributeValue operator()(nostd::string_view v)` should accept `const char *s` as argument. We should be good removing `OwnedAttributeValue operator()(const char *s)`.

Reply:
Gotcha, removed this method